core/src/main/scala/kafka/server/ReplicaFetcherManager.scala

```diff
@@ -35,6 +35,11 @@ class ReplicaFetcherManager(brokerConfig: KafkaConfig,
   override def createFetcherThread(fetcherId: Int, sourceBroker: BrokerEndPoint): ReplicaFetcherThread = {
     val prefix = threadNamePrefix.map(tp => s"$tp:").getOrElse("")
     val threadName = s"${prefix}ReplicaFetcherThread-$fetcherId-${sourceBroker.id}"
+    try {
```

Comment:
Did you intend to include this change in the patch?

Reply:
Thanks for the review. I don't intend to include this change.
It's put there to troubleshoot why the tests are still failing in CI, but not on my local box. I'll put the PR to draft mode during the troubleshooting.